opensubtitles/download.go

```diff
@@ -0,0 +1,59 @@
+package opensubtitles
+
+import (
+	"context"
+	"net/http"
+)
+
+type DownloadService service
+
+type DownloadOptions struct {
+	//ID of the file to download
+	FileID int `url:"file_id,omitempty"`
+	//Format (optional, default to original. possible values: srt, sub, mpl, webvtt, dfxp, txt)
+	SubFormat string `url:"sub_format,omitempty"`
+	//Desired name of the returned file
+	FileName string `url:"file_name,omitempty"`
+	//Remove HTML tags (default false)
+	StripHTML string `url:"strip_html,omitempty"`
+	//Remove HTML links
+	CleanupLinks string `url:"cleanup_links,omitempty"`
+	//Remove ads
+	RemoveAds string `url:"remove_adds,omitempty"`
+	//Input FPS (advanced, default to original subtitle FPS)
+	InFPS string `url:"in_fps,omitempty"`
+	//Output FPS
+	OutFPS string `url:"out_fps,omitempty"`
+	//Timeshift (+/- time in ms or s, eg +2s or -200ms)
+	Timeshift string `url:"timeshift,omitempty"`
+}
+
+type Download struct {
```

Comment:
Codacy found an issue: [exported type Download should have comment or be unexported](https://app.codacy.com/manual/TheForgotten69/go-opensubtitles/pullRequest?prid=6093402)